automation/jenkins_build.sh

```diff
@@ -1,6 +1,22 @@
 #!/bin/bash
 set -e
 
+BUILD_CONTAINER_NAME=yocto-build-$$
+
+cleanup() {
+    echo "[INFO] jenkins_build.sh: Cleanup."
+
+    # Stop docker container
+    echo "[INFO] jenkins_build.sh: Cleaning up yocto-build container."
+    docker stop $BUILD_CONTAINER_NAME 2> /dev/null || true
+    docker rm --volumes $BUILD_CONTAINER_NAME 2> /dev/null || true
+
+    if [ "$1" == "fail" ]; then
+        exit 1
+    fi
+}
+trap 'cleanup fail' SIGINT SIGKILL
```

Comment:
Good point. That was actually a typo initially. For whatever reason it remained here. I'll fix.